lib/modules/home/utils/onboarding.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:andax/models/story.dart';
+import 'package:andax/store.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+
+Future<StoryInfo?> needsOnboarding() async {
+  final prefs = await SharedPreferences.getInstance();
+  final onboarded = prefs.getBool('onboarded') ?? false;
+  if (onboarded) return null;
+
+  final obj = await algolia
+      .index('stories')
+      .getObjectsByIds(['STXZlBw4mA3culnp0tYi'])
```

Comment:
Magic constant. Better move it to a `const` value. Also, couldn't its ID be `onboarding`?

Reply:
I'd like to have 'onboarding' ID, but not sure how to change the Firebase ID without manual move.